configure.ac

```diff
@@ -85,6 +85,9 @@ LT_LIB_M
 AC_C_RESTRICT
 AC_FUNC_STRERROR_R
 
+LIBTOOL_NO_SUPPRESS="-no-suppress"
```

Comment:
rename it to LT_CPPFLAGS and define it along with CPPFLAGS in compiler.m4

Reply:
Perhaps `_CFLAGS` (as `CPPFLAGS` are initialized from `CFLAGS` in other places, not the opposite).

Reply:
Renamed `LIBTOOL_NO_SUPPRESS` to `LT_CFLAGS`
Moved the definition from `configure.ac` to `compiler.m4`
@yosefe @gleon99